README.md

```diff
@@ -1,2 +1,30 @@
 # PalArt-Gallery
```

Comment:
this is how the readme looks, you need to restyle it

![Screenshot from 2021-03-04 23-47-26](https://user-images.githubusercontent.com/47992412/110035701-a71ee500-7d44-11eb-840d-f832a13c4df6.png)